pyanaconda/payload/rpmostreepayload.py

```diff
@@ -53,6 +54,10 @@ def __init__(self, *args, **kwargs):
         self._internal_mounts = []
         self._locale_map = None
 
+        self._flatpak_payload = FlatpakPayload(conf.target.system_root)
+        # Initialize temporal repo to enable reading of the remote
+        self._flatpak_payload.initialize_with_path("/var/tmp/anaconda-flatpak-temp")
```

Comment:
Is it safe to have FlatpackPayload (even initialized) in RPMOSTreePayload by default? I mean from the perspective of use cases without flatpacks - now running code (and hitting potential bugs) which is not necessary to run.
But maybe we don't care at this point.
And shouldn't we check if the payload is available before trying to initialize it?

Reply:
Good point maybe it would make sense to promote the `is_available` as a static method.

Reply:
Fixed by checking is available before creation.